contracts/utils/Executor.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.5 <0.9.0;
+
+// solhint-disable no-inline-assembly
+
+library Executor {
```

Comment:
Is there a library like that in, say, OpenZeppelin collection?

Reply:
there is, but it seems gas-innefficient.. this one is taken from Gnosis.